### PR TITLE
Psg 4900 use repo templates

### PR DESCRIPTION
**Pull request by jenn-macfarlane**
## What's New?

Uses repo templates to format README, LICENSE, and remove community files in favour of using organization community files.

## Screenshots (if appropriate):

CleanShot 2024-10-22 at 12 21 33@2x

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have manually tested my code thoroughly
- [x] I have added/updated inline documentation for public facing interfaces if relevant
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing integration and unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Additional context

Add any other context or screenshots about the pull request here.

### CODE REVIEW

File: PassageFlex.podspec [hunk 1]
```diff
@@ -1,15 +1,18 @@
 Pod::Spec.new do |s|
   s.name             = 'PassageFlex'
   s.version          = '0.2.0'
-  s.summary          = 'Use Passage Flex in your iOS application'
-  s.homepage         = 'https://github.com/passageidentity/passage-flex-ios'
-  s.license          = { :type => 'MIT', :file => 'LICENSE.md' }
-  s.author           = { 'Passage Identity, Inc' => 'hello@passage.id' }
+  s.summary          = 'Passkey Flex for Apple Platforms - Add native passkey authentication to your own Swift authentication flows with Passage by 1Password'
```
Comment (SinaSeylani) [inline PassageFlex.podspec:4]: this should be iOS instead of Apple Platforms.

File: PassageFlex.podspec [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
+  s.description      = <<-DESC
+Passkey Flex for Apple Platforms - Add native passkey authentication to your own Swift authentication flows with Passage by 1Password
```
Comment (SinaSeylani) [inline PassageFlex.podspec:6]: same here

File: README.md [hunk 3]
```diff
@@ -1,46 +1,87 @@
...
+
+Use [Passkey Flex](https://docs.passage.id/flex) to add passkeys to an existing authentication experience.
+
+Use [Passkey Complete](https://docs.passage.id/complete) as a standalone passwordless auth solution.
+
+Use [Passkey Ready](https://docs.passage.id/passkey-ready) to determine if your users are ready for passkeys.
+
+### In passage-flex-ios
+
+Use passage-swift to implement Passkey Flex in your Swift application to add native passkey authentication in your own authentication flows.
```
Comment (SinaSeylani) [inline README.md:20]: I think this is wrong. we use passage-flex-ios in only iOS Applications, and passage-swift for all Apple platforms. @rickycpadilla, can you please confirm this?

Comment (jenn-macfarlane) [reply README.md:20]: @SinaSeylani are we waiting for confirmation for these?

Comment (jenn-macfarlane) [reply README.md:20]: I've updated the repo description to reflect this, do these tags need to be updated/any removed? I imagine macOS/tvos/watchos might need to go?

swift authentication passkeys passwordless webauthn apple ios macos passkey-flex tvos watchos biometrics 1password passage passage-sdk

Comment (agebhardt) [reply README.md]: I believe passage-flex-ios _does_ support MacOS and tvOS:
https://github.com/passageidentity/passage-flex-ios/blob/main/Package.swift#L8

This was one of those naming things we haven't changed yet (though we could) to better reflect the support.

@rickycpadilla can you confirm my assumptions about what passage-flex-ios supports?

Comment (SinaSeylani) [reply README.md]: @jenn-macfarlane  @agebhardt  yes that's correct
passage-flex-ios ios, macos, tvos
passage-swift supports All the Apple platforms

Comment (jenn-macfarlane) [reply README.md]: Ok, so what's the recommendation here? I need to confirm what we want for the following:

**Repo description:** 

`Passkey Flex for iOS - Add native passkey authentication to your own Swift authentication flows with Passage by 1Password`

**Tags/keywords:**

- swift
- authentication
- passkeys
- passwordless
- webauthn
- apple
- ios
- macos
- passkey-flex
- tvos
- watchos
- biometrics
- 1password
- passage
- passage-sdk

Comment (agebhardt) [reply README.md]: I'd like to recommend we move forward with the original guidance here, @jenn-macfarlane -- while "Apple platforms" is slightly more generic, I think it reads better and offers some simplicity. I don't anticipate problems for folks.

"ios, macos, and tvos" are all apple platforms
while there are _more_ apple platforms (like what passage-swift supports), I'd like to nudge us to use similar language.

Comment (jenn-macfarlane) [reply README.md]: Maybe a bit pedantic but "ios, macos, and tvos" does not include watchOS so I wanted to confirm.

I'm good to go with the OG guidance, meaning:

`Passkey Complete for Apple Platforms - Go completely passwordless with a standalone auth solution in your Swift apps with Passage by 1Password`

and the list of keywords above unchanged.

Comment (agebhardt) [reply README.md]: How about:

`Passkey Flex for Apple Platforms - Go completely passwordless with a standalone auth solution in your iOS, MacOS, and tvOS apps with Passage by 1Password`

Comment (agebhardt) [reply README.md]: Let's remove `watchos` from the list of keywords

Comment (jenn-macfarlane) [reply README.md]: Done!

**Comment by jenn-macfarlane** [comment]
@agebhardt @SinaSeylani are we good with these changes?

**Review by SinaSeylani — APPROVED** [review]

**Pull request merged**